paper-drawer-panel.html

```diff
@@ -265,6 +269,10 @@
 
     'use strict';
 
+   // this is the only `paper-drawer-panel` in
+   // the whole app that can be in `dragging` state
+    var sharedPanel = null;
```

Comment:
`sharedPanel` is private in the sense that it's in a closure (IIFE). They aren't accesible to the external world.